python/spec/dependabot/python/file_parser_spec.rb

```diff
@@ -65,6 +65,29 @@
       its(:length) { is_expected.to eq(3) }
     end
 
+    context "with jinja templates" do
+      let(:requirements_fixture_name) { "jinja_requirements.txt" }
+      describe "the first dependency" do
+        subject(:dependency) { dependencies.first }
+
+        it "has the right details" do
+          dependency = dependencies.first
+          expect(dependency).to be_a(Dependabot::Dependency)
```

Comment:
Don't think we need to check the type here if you're checking the `name` `version` attributes
```suggestion
```